django/chat/views.py

```diff
@@ -366,26 +366,29 @@ def chunk_upload(request, message_id):
 
     file = request.FILES["file"].read()
     content_type = request.POST["content_type"]
-    fileName = request.POST["filename"]
+    file_name = request.POST["filename"]
     file_id = request.POST["file_id"]
     end = request.POST["end"]
     nextSlice = request.POST["nextSlice"]
 
-    if file == "" or fileName == "" or file_id == "" or end == "" or nextSlice == "":
+    if ("text" in content_type or not content_type) and file_name.endswith(".md"):
```

Comment:
Markdown content type is not usually detected. We will just assume it is markdown if file is plain text and ends in "md".